Source/WebCore/accessibility/cocoa/AccessibilityObjectCocoa.mm

```diff
@@ -196,7 +196,7 @@ static void attributedStringAppendWrapper(NSMutableAttributedString *attrString,
     auto result = adoptNS([[NSMutableArray alloc] init]);
 
     // Iterate over the range to build the AX attributed strings.
-    for (TextIterator it(range); !it.atEnd(); it.advance()) {
+    for (TextIterator it(range, { TextIteratorBehavior::IgnoresFullSizeKana }); !it.atEnd(); it.advance()) {
         Node& node = it.range().start.container;
 
         // Non-zero length means textual node, zero length means replaced node (AKA "attachments" in AX).
```

Comment:
Maybe we can add a helper function that returns a TextIterator with the behaviors we want instead of calling the constructor directly.
Also, are we covering all the TextIterators we use? e.g., the ones used to build AttributedStrings?

Reply:
I can definitely add a helper to make this cleaner.

For your second point, this covers all places where we return strings (both attributed and regular). AttributedStrings are created using `contentForRange`, which we now use this new TextIteratorBehavior in. We don't need to use this behavior when computing text markers with iterators, however, since this only affects character appearance.